docs/pages/docs/guide/built-ins/table.mdx

```diff
@@ -0,0 +1,52 @@
+import { Table, Td, Th, Tr } from 'nextra/components'
+
+# Table Component
+
+A collection of built-in components designed to create styled, non-markdown
+(i.e., literal) HTML tables.
```

Comment:
I explicitly mentioned `styled` here, wdyt?

Reply:
Looks good to me! Just make the changes as you see fit, totally fine with me. I'm just happy to be able to contribute 😊

Reply:
thanks, anyway I prefer to know your opinion after my applied changes!

Reply:
Off topic here: do you know where you get your curated icons from? Initially I couldn't find the TableIcons and was very close to rolling my own... 

Just asking so that if I'm making a PR in the future that requires a new icon, I can make sure it's up to the design specs established for Nextra 😊

Reply:
I usually take a look at https://www.radix-ui.com/icons first, if I don't find here I just search in google some free svg icons